package.json

```diff
@@ -22,11 +22,12 @@
     "promote": "heroku pipelines:promote -r staging",
     "dump-schema": "babel-node ./scripts/dump-schema.js",
     "heroku-postbuild": "babel index.js config.js -s inline -d build && babel lib -s inline -d build/lib && babel schema -s inline -d build/schema",
-    "pre-commit": "lint-staged"
+    "precommit": "lint-staged"
```

Comment:
this was a typo before, now works:

![screen shot 2017-05-01 at 14 00 48](https://cloud.githubusercontent.com/assets/49038/25580183/9b33fc24-2e76-11e7-9cef-264d8b505bd2.png)